common/scripted_triggers/wc_scripted_artifacts_triggers.txt

```diff
@@ -28,4 +30,155 @@ crossbow_artifact_active_trigger = {
 
 rifle_artifact_active_trigger = {
 	location = { TECH_SIEGE_EQUIPMENT = 4 }
+}
+
+# Can get at gamestart
+
+can_get_amulet_at_gamestart_distribution_trigger = {
+	NOT = { any_artifact = { has_artifact_flag = amulet } }
+}
+
+can_get_ring_at_gamestart_distribution_trigger = {
+	NOT = {
+		any_artifact = {
+			has_artifact_flag = ring
+			count = 2
+		}
+	}
+}
+
+can_get_bracelet_at_gamestart_distribution_trigger = {
+	NOT = { any_artifact = { has_artifact_flag = bracelet } }
+}
+
+can_get_weapon_at_gamestart_distribution_trigger = {
+	NOT = { any_artifact = { has_artifact_flag = combat_weapon } }
+}
+
+can_get_melee_weapon_at_gamestart_distribution_trigger = {
+	can_get_weapon_at_gamestart_distribution_trigger = yes
+	is_theocracy = no # Classless priests receive artifacts after game start and instantly gain classes, making artifacts inactive
+	shared_physical_weapon_trigger = yes
+}
+
+can_get_ranged_weapon_at_gamestart_distribution_trigger = {
+	can_get_weapon_at_gamestart_distribution_trigger = yes
+	is_theocracy = no # Classless priests receive artifacts after game start and instantly gain classes, making artifacts inactive
+	shared_ranged_weapon_trigger = yes
+}
+
+can_get_artifacts_at_gamestart_distribution_trigger = {
+	is_landed = yes
+	OR = {
+		can_get_amulet_at_gamestart_distribution_trigger = yes
+		can_get_ring_at_gamestart_distribution_trigger = yes
+		can_get_bracelet_at_gamestart_distribution_trigger = yes
+		can_get_melee_weapon_at_gamestart_distribution_trigger = yes
+		can_get_ranged_weapon_at_gamestart_distribution_trigger = yes
+	}
+}
+
+# Preferences
+
+prefers_melee_weapons_trigger = {
+	OR = {
+		is_warrior_class_trigger = yes
+		is_paladin_class_trigger = yes
+		is_rogue_class_trigger = yes
+		is_death_knight_class_trigger = yes
+	}
+}
+
+prefers_ranged_weapons_trigger = {
+	# OR = {
+		is_hunter_class_trigger = yes
+	# }
+}
+
+prefers_onehanded_weapons_trigger = {
+	# OR = {
+		is_small_humanoid_trigger = yes
+	# }
+}
+
+prefers_twohanded_weapons_trigger = {
+	# OR = {
+		is_large_humanoid_trigger = yes
+	# }
+}
+
+prefers_swords_trigger = {
```

Comment:
There's `use_scimitars_trigger` and BASICALLY these cultures use swords.

Reply:
Done